lib/iris/tests/results/file_load/known_loaders.txt

```diff
@@ -1,16 +1,16 @@
  * NetCDF OPeNDAP (priority 6)
  * NAME III (priority 5)
- * NetCDF_v4 (priority 5)
- * UM Post Processing file (PP) (priority 5)
  * NetCDF (priority 5)
  * NetCDF 64 bit offset format (priority 5)
+ * NetCDF_v4 (priority 5)
+ * UM Post Processing file (PP) (priority 5)
```

Comment:
Format specifications with the same priority *should* have mutually exclusive file matching criteria, in which case we would be able to get away with it. I'll check...